context/MqlContext/MqlContextProvider.tsx

```diff
@@ -103,15 +110,18 @@ function MqlContextProviderInternal({
       // Note: newServerid is cast as a string because it is stored on the backend
       // as a User Preference, whose values are all strings.
       const userId = getUserIdFromAuthToken(token);
+      const refetch = externalConfig
+        ? externalConfig?.refetchMqlServerUrl
+        : refetchMqlServerUrl;
```

Comment:
Should we log an error if this gets hit? Now that the only client of this is our UI, theoretically there will always be an external config right?

Reply:
In theory, yeah.

Reply:
I'll see if I can think of the right way to do this. :)